extensions/amp-animation/0.1/web-animation-types.js

```diff
@@ -168,6 +168,8 @@ const WHITELISTED_RPOPS = {
   'opacity': true,
   'transform': true,
   'visibility': true,
+  'offset-distance': true,
```

Comment:
I'll go ahead and merge. It's possible we'll convert properties in upcoming tasks.